lib/matplotlib/tests/test_axes.py

```diff
@@ -468,6 +469,12 @@ def test_inverted_cla():
     plt.close(fig)
 
 
+def test_cla_not_redefined():
+    for klass in Axis.__subclasses__():
+        # check that subclasses do not get redefined in our Axis subclasses
```

Comment:
```suggestion
    for klass in Axes.__subclasses__():
        # check that subclasses do not get redefined in our Axes subclasses
```
I think you want the "e" Axes instead of Axis?